packages/vaex-core/vaex/test/dataset.py

```diff
@@ -2161,6 +2161,31 @@ def test_lasso(self):
 		self.assertAlmostEqual(sumx, 0+1+2)
 		self.assertAlmostEqual(sumy, 0+1+4)
 
+	def test_select_circle(self):
+		# Generate test data
+		x = np.arange(-2,3)
+		x, y = np.meshgrid(x,x)
+
+		# add to dataset
+		ds = vx.from_arrays(x=x, y=y)
+		# do the selection
+		ds.select_circle('x', 'y', 0.5, 0.5, 1, name='circ')
```

Comment:
pls use self.dataset, there are deliberate 'quircks' in them, such as missing data etc. It also has x and y.